tpcx_bb/benchmark_runner.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
We should update this to have repeated runs as well as print for the current query and the garbage collection bit. 

https://gist.github.com/VibhuJawa/3df91fff23917be66b30f56e5fdf2556

Reply:
Updated